assets/page.html

```diff
@@ -3,106 +3,9 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="bootstrap.min.css" media="screen">
+    <link rel="stylesheet" href="styles.css" media="screen">
```

Comment:
If this is going to be moved to a separate file it needs a `?vary=#` on it (this is a way to workaround caching issues).